src/components/components/leader-tools/leader-tools.js

```diff
@@ -33,7 +33,13 @@ const LeaderTool = props => {
 
   return (
     <li className="leader-tool">
-      <a href={url} className="leader-tool__link" title={title}>
+      <a
```

Comment:
I don't remember if this is an external link or not, could you verify that this is an external link?

Reply:
yes sure

Reply:
this is an external link so we need that here